src/NetAnalyzers/UnitTests/Microsoft.CodeQuality.Analyzers/Maintainability/ReviewUnusedParametersTests.Fixer.cs

```diff
@@ -385,7 +385,7 @@ public void Caller()
             var fix = @"
 class C
 {
-    public static explicit operator int(C [|value|]) => 0;
+    public static explicit operator int(C _) => 0;
```

Comment:
Does the test library support selecting different code actions in the same test?